### PR TITLE
Feature/7/remove dashboard widgets

### PR DESCRIPTION
**Pull request by vancoder**
## Summary

Relates to https://github.com/alleyinteractive/wp-alleyvate/issues/7

## Notes for reviewers

None.

### CODE REVIEW

File: src/alley/wp/alleyvate/features/class-dashboard-widget-removal.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		}
+	}
+
+	/**
+	 * Getter for widgets to be removed.
+	 *
+	 * @return string[][]
+	 */
+	public function get_widgets() {
+		return [
```
Comment (attackant) [inline src/alley/wp/alleyvate/features/class-dashboard-widget-removal.php:48]: Would it make more sense to define the array of widgets as a property?

**Review by attackant — APPROVED** [review]
Left a comment but looks good.

**Review by kevinfodness — CHANGES_REQUESTED** [review]
Left some comments related to the tests. Implementation of the feature itself is solid.

File: tests/alley/wp/alleyvate/features/test-dashboard-widget-removal.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		global $wp_meta_boxes;
+
+		$array_keys = $this->array_keys_r( $wp_meta_boxes );
+
+		foreach ( $this->feature->get_widgets() as $widget ) {
+			$this->assertNotContains(
+				$widget['id'],
+				$array_keys,
+				$widget['id'] . ' was not removed from dashboard widgets.'
+			);
```
[the remark this answers is not in the document]
Comment (vancoder) [reply tests/alley/wp/alleyvate/features/test-dashboard-widget-removal.php:65]: If the feature has worked, it has removed the specified element from the array - therefore this suggested method gets an `Undefined array key "dashboard_primary"` error. Am I missing something?

**Review by kevinfodness — APPROVED** [review]
🍣

**Pull request merged**